data/json/items/armor/torso_clothes.json

```
@@ -980,7 +980,7 @@
       {
         "id": "mouse_portal_tshirt",
         "name": { "str": "mouse portal t-shirt" },
-        "description": "A short woth blue portal with a mouse head sticking out of it.  The mouse is holding some kind of gunlike device towards a cube.  Is the cube made of cheese?  The back of the shirt has a yellow portal with the back half of the mouse entering the portal.",
+        "description": "A shirt with blue portal with a mouse head sticking out of it.  The mouse is holding some kind of gun-like device towards a cube.  Is the cube made of cheese?  The back of the shirt has a yellow portal with the back half of the mouse entering the portal.",
```

```suggestion
        "description": "There is a blue portal on this shirt, with a mouse head sticking out of it.  The mouse is holding some kind of gun-like device towards a cube.  Is the cube made of cheese?  The back of the shirt has a yellow portal with the back half of the mouse entering the portal.",
```

My initial thought would be to add "a" here: "a shirt with a blue portal", which would also work, but I think this reads better and is similar to the other descriptions. Feel free to rephrase it or change the order however you want though.